src/components/form-elements/checkbox/Checkbox.stories.mdx

```diff
@@ -432,6 +434,69 @@ In the following example, className `sg-checkbox--custom-theme` was applied to a
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story name="With controlled checkboxes">
+    {args => {
+      const dataObject = {
+        101: {
+          id: 101,
+          value: true,
+          name: 'I receive a new answer to my question',
+        },
+        102: {
+          id: 102,
+          value: true,
+          name: 'My answer is chosen as the best',
+        },
+        103: {
+          id: 103,
+          value: true,
+          name: 'I receive a friend request',
+        },
+        104: {
+          id: 104,
+          value: true,
+          name: 'Somebody thanks me for my help',
+        },
+        105: {
+          id: 105,
+          value: true,
+          name: 'There are new comments to my questions',
+        },
+        106: {
+          id: 106,
+          value: true,
+          name: 'There are new comments to my answers',
+        },
+      };
+      const [data, setData] = useState(dataObject);
+      const onChange = id => {
+        const newData = {...data};
+        newData[id] = {...data[id], value: !data[id].value};
+        setData(newData);
+      };
+      return (
+        <Flex direction="column">
+          <div>
+            <Button onClick={() => onChange(101)} size="s">
+              click to toggle first checkbox
+            </Button>
```

Comment:
I think this button is unnecessary - you can just click the first checkbox :D

Reply:
@katarzynatobis it is to change the active checkbox without clicking it